vocab/quantitykinds/VOCAB_QUDT-QUANTITY-KINDS-ALL-v2.1.ttl

```diff
@@ -18676,3 +18676,11 @@ quantitykind:WarpingMoment
     rdfs:label "Warping Moment"@en ;
 .
 
+quantitykind:FrictionLoss
+    a qudt:QuantityKind ;
+    qudt:hasDimensionVector qkdv:A0E0L-2I0M1H0T-2D0;
```

Comment:
The dimension seems wrong. If you really mean "power", then the dimension should be A0E0L2I0M1H0T-3D0. If you mean energy, then it would be A0E0L2I0M1H0T-2D0

Reply:
You might also want to include a skos:broader statement.

Reply:
The unit associated with the name 'friction loss' in the IFC model I was working on was kg / (m²s²), that's where I took the dimensions from. In the wikipedia article, https://en.wikipedia.org/wiki/Friction_loss, this unit is referred to as 'friction loss per unit length'. I had to add that unit to the units file as well. 

I took the description from the wikipedia article, which covers friction loss in a number of different fields. I may have gotten the wrong one, given that the unit kg/(m²s²) appears in the section on 'Calculating friction loss for air in a duct'. Here are the 3 definitions of 'friction loss' (I used the second, but the first seems better suited):
 
> The term friction loss (or frictional loss) has a number of different meanings, depending on its context.
>
> In [fluid flow](https://en.wikipedia.org/wiki/Fluid_flow) it is the [head loss](https://en.wikipedia.org/wiki/Head_loss) that  occurs in a containment such as a pipe or duct due to the effect of the fluid's [viscosity](https://en.wikipedia.org/wiki/Viscosity) near the surface of the containment.[[1]](https://en.wikipedia.org/wiki/Friction_loss#cite_note-Munson-1)

> In mechanical systems such as [internal combustion engines](https://en.wikipedia.org/wiki/Internal_combustion_engine), the term refers to the power lost in overcoming the [friction](https://en.wikipedia.org/wiki/Friction) between two moving surfaces. 

> In economics, frictional loss is natural and irrecoverable loss in a transaction or the cost(s) of doing business too small to account for. Contrast with [tret](https://en.wikipedia.org/wiki/Trett) in shipping, which made a general allowance for otherwise unaccounted for factors.

How about we rename the quantity kind to `qk:FrictionLossPerLength`, use the first definition and keep the dimension vector? If there is a good candidate for a broader candidate, I did not find it. Help appreciated!

Reply:
Actually, how about we make the URI qk:PressureLossPerLength, which is actually more correct, and then in the description we can say that this is sometimes referred to as "Friction Loss?"

My reasoning is that "friction" is actually a force, while "pressure" is a force per area. The Wikipedia article you mention shows the table as a delta (Pressure) / Length.

Reply:
ok, done.